ci/travis/38-latest-conda-forge.yaml

```diff
@@ -0,0 +1,25 @@
+name: test
+channels:
+  - conda-forge
+dependencies:
+  - python=3.8
+  # required
+  - pandas
+  - shapely
+  - fiona
+  - pyproj
+  # testing
+  - pytest
+  - pytest-cov
+  - codecov
+  # optional
+  - rtree
+  - matplotlib
+  - descartes
+  - pysal
+  - geopy
+  - SQLalchemy
+  - psycopg2
+  - libspatialite
+  # TEMP: workaround until conda-forge fixes the issues with gdal
```

Comment:
Is the workaround still necessary in the meantime?

Reply:
Locally not. Trying on Travis now.